packages/host/app/components/operator-mode/container.gts

```diff
@@ -349,34 +349,21 @@ export default class OperatorModeContainer extends Component<Signature> {
   @action onCardSelectFromSearch(card: Card) {
     let searchSheetTrigger = this.searchSheetTrigger; // Will be set by onFocusSearchInput
 
-    if (!searchSheetTrigger) {
-      throw new Error('bug: searchSheetTrigger should be set here');
-    }
-
     // This logic assumes there is currently one stack when this method is called (i.e. the stack with index 0)
 
     // In case the left button was clicked, whatever is currently in stack with index 0 will be moved to stack with index 1,
-    // and the card will be added to stack with index 0.
+    // and the card will be added to stack with index 0. moveStack executes this logic.
     if (
       searchSheetTrigger ===
       SearchSheetTrigger.DropCardToLeftNeighborStackButton
     ) {
+      this.operatorModeStateService.moveStack(0);
```

Comment:
@habdelra mentioned in passing that the `replaceItemInStack` function was doing two things. One which is replacing and another which is shifting the stack as shown [here](https://github.com/cardstack/boxel/blob/e528293f793895d4cb9428a1cd959568cf17cd68/packages/host/app/services/operator-mode-state-service.ts#L64-L76), 

This function doesn't separate the exact function out item-per-item rather it moves entire stack.